app/hook/storage.ts

```diff
@@ -52,9 +52,9 @@ export const useTransactionLogs = (accountId: string) => {
   })
 }
 
-export const usePendingRequests = () => {
+export const useRequests = () => {
   return useStorage(({ state }) => {
-    return Object.values(state.pendingRequest)
+    return Object.values(state.request)
```

Comment:
I'm thinking in App side, every tx in `Sent` status should be filtered out because our app will stay in Review page unless the tx has been resolved.

Reply:
Not sure if I understand the question correctly. 

In `state.request` we would only have `TransactionRequest`, transaction with status `Sent` would exist in `state.requestLog`, so we should not need to do the filter here, right?

Reply:
You're right! we don't need to do the filter here.  🫣
Thanks for helping me understanding the code.